packages/fiori-elements-writer/package.json

```diff
@@ -11,7 +11,6 @@
         "lint:fix": "eslint . --ext .ts --fix",
         "test": "jest --ci --forceExit --detectOpenHandles --colors",
         "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
-        "test:local": "node -r ts-node/register test/index.ts",
```

Comment:
Thanks for cleaning this up. What was this script meant to be test? Can't recall now.

Reply:
That was one of my old scripts that we removed when publishing the elements templates.